### ISSUE TITLE
Cannot drop pins on map

### ISSUE THREAD

**Issue by kmeyersvt** [codeforbtv/green-up-app-old#222]
**Describe the bug**
No matter where I tap on the Map, it doesn't let me drop a  pin.

**To Reproduce**
1. Click the Trash icon.
2. Zoom in  to find a  street.
3. Tap on  the map to drop a pin.
4. Nothing happens.
5. Tap "Record your trash", and the app crashes

**Expected behavior**
I expect a pin to appear on the map.

**Smartphone (please complete the following information):**
 - Device: Samsung S9
 - Version Android v10

**Comment by mpettit** [comment]
Looks like the use case doesn't quite reflect what we have in the code so far.  Are users selecting a single new drop point on the map or are they selecting one of the other collection pins that already show on the map?

**Comment by smara-codes** [comment]
I submitted a fix for this issue in PR #259. Here's how the app will behave:

1. Go to Trash screen
2. Tap Record Your Trash
3. Tap Save

Now see trash drop on the map. Part of this PR I also fixed the display of trash drops on the map. My trash will show up in yellow pins while other's, uncollected trash is in red pins.

Poking around it seems that the collect trash functionality is no longer available.
